Add enterprise login feature documentation (#525)
wso2/docs-choreo-dev: en/docs/references/enterprise-login.md

# Choreo Enterprise Login

Choreo's enterprise login feature allows your users residing in an external IdP (Identity Provider) to login into Choreo seamlessly without changing their credentials.

This guide takes you through the steps you need to follow to configure an enterprise login for your organization in Choreo. 

Let's begin...

## Step 1: Create an organization in Choreo

1. Sign in to the Choreo Console at <a>https://console.choreo.dev/</a>. using a Google / GitHub / Microsoft account.
2. If you are a:
    1. **New user**: enter a unique organization name and create an organization. For example, "Stark Industries".

        ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
 
    2. **Returning user**: expand the drop-down for your profile and click **Settings**. Under **Organizations**, you can view the organization you created at sign-up. 
 
## Step 2: Configure enterprise login for your Choreo organization

1. Expand the drop-down for your profile and click **Settings**.

> 2. Click the **Copy Handle** button to copy the organization handler into the clipboard.

Reviewer: ```suggestion
2. Click the **Copy Handle** button to copy the organization handle into the clipboard.
```

     ![Copy organization name](../assets/img/references/enterprise-login/copy-organization-name.png){.cInlineImage-half}

3. Next, As Choreo reserves the organization name to that specific user account, we need to create an organization on the Choreo IdP(Asgardeo) with the same name. Sign up to Asgardeo <a>https://asgardeo.io/signup?utm_source=console</a> using the credentials that was used to register to Choreo. 

4. Paste the value copied in step 2 as the organization name in Asgardeo and click **Create**. 
     ![Create an organization in Asgardeo](../assets/img/references/enterprise-login/create-asgardeo-organization.png){.cInlineImage-half}

5. Finally, send an email to `choreo-help@wso2.com` requesting to configure enterprise login to your organization. 
 Mention the following information in the request:
   1. Organization name or handle. For example, “Stark Industries” or “starkindustries”
   2. Email domains specific to your organization. For example, “@stark.com”, “@starkindustries.com”, and “@stark.eu”.

> 

Reviewer: Should we add a sample email to give an idea about the template to the customer?
One is added in the doc
https://docs.google.com/document/d/1bz4vdAy-vyrpEa90Yz3DOOfibKRd9XP_cKEiwvx2Qwg/edit?usp=sharing

Author: +1

# Enterprise Login

> Choreo's enterprise login feature allows your users residing in an external IdP (Identity Provider) to login into Choreo seamlessly without changing their credentials.

Reviewer 2: Since we use the word "feature", shall we have "Enterprise Login" in title caps?

Author: done

This guide takes you through the steps you need to follow to configure an enterprise login for your organization in Choreo. 

Let's begin...

## Step 1: Create an organization in Choreo

> 1. Sign in to the Choreo Console at <a>https://console.choreo.dev/</a>. using a Google / GitHub / Microsoft account.

Reviewer 2: ```suggestion
Sign in to the Choreo Console at <a>https://console.choreo.dev/</a> using a Google / GitHub / Microsoft account.

If you are a:

- **New user**: enter a unique organization name and create an organization. e.g., `Stark Industries`.
- **Returning user**: expand the drop-down for your profile and click **Settings**. Under **Organizations**, you can view the organization you created at sign-up.
```

Author: Done

2. If you are a:

>     1. **New user**: enter a unique organization name and create an organization. For example, "Stark Industries".

Author: Used bullet points.

        ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
 
    2. **Returning user**: expand the drop-down for your profile and click **Settings**. Under **Organizations**, you can view the organization you created at sign-up. 
 
## Step 2: Configure enterprise login for your Choreo organization

> 

Author: Adopted the suggestion with changes.

1. Expand the drop-down for your profile and click **Settings**.

> 2. Click the **Copy Handle** button to copy the organization handle into the clipboard.

Author: Done.

     ![Copy organization name](../assets/img/references/enterprise-login/copy-organization-name.png){.cInlineImage-half}

> 3. Next, As Choreo reserves the organization name to that specific user account, we need to create an organization on the Choreo IdP(Asgardeo) with the same name. Sign up to Asgardeo <a>https://asgardeo.io/signup?utm_source=console</a> using the credentials that was used to register to Choreo. 

Author: We need to give this information out. As it is an admin who will be using it.

> 3. Next, As Choreo reserves the organization name to that specific user account, we need to create an organization on the Choreo IdP(Asgardeo) with the same name. Sign up to Asgardeo <a>https://asgardeo.io/signup?utm_source=console</a> using the credentials that was used to register to Choreo. 

Author: Applied with changes.

4. Paste the value copied in step 2 as the organization name in Asgardeo and click **Create**. 
     ![Create an organization in Asgardeo](../assets/img/references/enterprise-login/create-asgardeo-organization.png){.cInlineImage-half}

5. Finally, send an email to `choreo-help@wso2.com` requesting to configure enterprise login to your organization. 
 Mention the following information in the request:
   1. Organization name or handle. For example, “Stark Industries” or “starkindustries”
   2. Email domains specific to your organization. For example, “@stark.com”, “@starkindustries.com”, and “@stark.eu”.

> !!!info "Email Sample"       

Author: done

              Subject : [Stark Industries] Configure enterprise login
              Content:
                     Hi CS team,
                     Configure enterprise login to my organization and please find the relevant information below.
                     Organization name or handle
                     Eg: “Stark Industries” or “starkindustries”

                     Email domains specific to your organization
                     Eg: “@stark.com”, “@starkindustries.com”, “@stark.eu”
                     Thank you!

That's it! 
Once our support team configures an enterprise login for your organization, you will receive an Email.

Next, you can create a connection to the federated identity provider (For example, Auth0, ADFS, Keycloak, etc.), following the steps below:

## Step 3: Bring your own identity

> Bring your own identity to Choreo by configuring a federated enterprise IdP on Asgardeo to your organization. Since you created the organization on Asgardeo with the same name you created on Choreo in [Step 2](#step-2-configure-enterprise-login-for-your-choreo-organization), Choreo can authenticate users signing in to that organization. Follow the steps below to configure the federated IdP:

Author: done

1. Sign in to Asgardeo at <a>https://asgardeo.io/signup?utm_source=console</a>.

> 2. Follow the steps in Asgardeo documentation on [Add Standard-Based Login](https://wso2.com/asgardeo/docs/guides/authentication/enterprise-login/) to configure a federated enterprise identity provider to your Asgardeo organization.

Reviewer 2: ```suggestion
2. To configure a federated enterprise identity provider to your Asgardeo organization, follow the steps in [Asgardeo Documentation - Add Standard-Based Login](https://wso2.com/asgardeo/docs/guides/authentication/enterprise-login/).
```

Author: done
